eq-author-api/docker-compose.yml

```diff
@@ -35,6 +35,7 @@ services:
       - JAEGER_SAMPLER_PARAM=1
       - REDIS_DOMAIN_NAME=redis
       - REDIS_PORT=6379
+      - ALLOWED_EMAIL_LIST=""
```

Comment:
This variable should be set; otherwise, we'll need to manually enter it to ensure behaviour matches the real thing, where it should be like this by default.